### PR TITLE
[vitotronic] Add channel "currentmode" to thing "circuit"

### PR DESCRIPTION
**Pull request by Fejitatete**
Signed-off-by: Fejitatete <fejitatete@msn.com>

### CODE REVIEW

**Comment by lolodomo** [comment]
A new thing channel with no Java code to manage it?

**Comment by jlaur** [comment]
See also #13735.

**Comment by Fejitatete** [comment]
Sry guys, I don't understand why this need some Java code. Could you explain or take a look in sources plz.
Thx

**Comment by Hilbrand** [comment]
@Fejitatete is right. It doesn't need code. The binding derives the channel from the data it receives from the device. It would be nice to add the channel to the readme. Also the channel seems to represent a mode. I suspect the mode numbers represent textuale description of what each number means. In that case you can add the option to the channel, so the user sees the textual description instead of just a number

File: bundles/org.openhab.binding.vitotronic/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -528,6 +529,13 @@
 		<state pattern="%d" readOnly="false" max="4" min="0" step="1"/>
 	</channel-type>
 
+	<channel-type id="currentmode">
+		<item-type>Number</item-type>
+		<label>Currentmode</label>
+		<description>Currentmode</description>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vitotronic/src/main/resources/OH-INF/thing/thing-types.xml:535]: Description to be removed if it adds no value to the label.
At least, use correct English, that is current mode (2 words).

File: bundles/org.openhab.binding.vitotronic/src/main/resources/OH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -528,6 +529,13 @@
...
+		<state pattern="%d" readOnly="true" max="3" min="0" step="1"/>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vitotronic/src/main/resources/OH-INF/thing/thing-types.xml:536]: As it is read only, min/max/step have no real interest, U believe,

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
The vitotronic.properties file has to be updated to include the new channel type and the 4 states.

New channel has to be also added in documentation (README.md file).

@Hilbrand : can you take the lead to review/merge this PR as you are apparently more aware than us how it is implemented ?

**Comment by Fejitatete** [comment]
> The vitotronic.properties file has to be updated to include the new channel type and the 4 states.
> 
> New channel has to be also added in documentation (README.md file).
> 
> @Hilbrand : can you take the lead to review/merge this PR as you are apparently more aware than us how it is implemented ?

**Comment by lolodomo** [comment]
No chance to have a proper signing ?

**Review by lolodomo — APPROVED** [review]
Thing definition and translation file are not in sync (label/description) for the new channel type.
I will merge as it is and provide a fix in a separate PR.

**Comment by lolodomo** [comment]
Making an exception and merge the PR even if not properly signed off.

**Pull request merged**